### PR TITLE
feat: create a separate 'create new library' page

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:** 
- Refactor create a new library popup on a separate page. 
- Replace all components with paragon components.
- Replace bootstrap styles with paragon styles.
- Add the ability to add custom branding via `@edx/brand` component.
- Add modal which shows a message about unsaving data

**Youtrack:**
https://youtrack.raccoongang.com/issue/EDX_BLND_CLI-191

**Coverage:**
  ```
  LibraryCreatePage.jsx                        |   84.81 |    65.85 |    87.5 |   84.21 | 71-77,83,154-157,182                                                        
  OrganizationDropdown.jsx                     |   95.77 |    71.79 |     100 |   95.31 | 45,96,107                                                                                                                             
  FormGroup.jsx                                |     100 |    66.66 |      80 |     100 | 8-36                                                                                                                                             
```

### CODE REVIEW

File: src/library-authoring/list-libraries/LibraryListPage.jsx [hunk 1]
```diff
@@ -50,16 +48,8 @@ export class LibraryListPage extends React.Component {
...
-      showForm: true,
-    });
-  }
-
-  hideForm = () => {
-    this.setState({
-      showForm: false,
-    });
+  goToCreateLibraryPage = () => {
+    this.props.history.push('/create');
```
Comment (ruzniaievdm) [inline src/library-authoring/list-libraries/LibraryListPage.jsx]: I think `history` needs be added to `LibraryListPage.propTypes`

File: src/vendor/studio.scss [hunk 2]
```diff
@@ -1845,3 +1886,42 @@ button {
...
+.dropdown-container {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.15), 0 2px 8px rgba(0, 0, 0, 0.15);
+  border-radius: 4px;
+  max-height: 200px;
+  font-size: 1rem;
+  font-weight: normal;
+  line-height: 1.25rem;
+  overflow-y: scroll;
+  position: absolute;
+  background-color: #fff;
```
Comment (ruzniaievdm) [inline src/vendor/studio.scss]: Please replace `#fff` to `$white` or something else suitable

File: src/vendor/studio.scss [hunk 3]
```diff
@@ -1845,3 +1886,42 @@ button {
...
+
+    .pgn__icon.btn-icon__icon {
+      height: 2.4rem;
+      width: 2.4rem;
+    }
+  }
+}
+
+.dropdown-container {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.15), 0 2px 8px rgba(0, 0, 0, 0.15);
```
Comment (ruzniaievdm) [inline src/vendor/studio.scss]: Please can you mapping needed box-shadow style on this [page](https://paragon-openedx.netlify.app/foundations/elevation)

File: src/vendor/studio.scss [hunk 4]
```diff
@@ -1845,3 +1886,42 @@ button {
...
+  border-radius: 4px;
```
Comment (ruzniaievdm) [inline src/vendor/studio.scss]: $baseline are preferred, can you change it, please

**Comment by ruzniaievdm** [comment]
@ihor-romaniuk great changes and tests! Left a couple of minor comments please take a look

**Review by ruzniaievdm — APPROVED** [review]

File: src/library-authoring/list-libraries/LibraryListPage.jsx (hunk 1 above)
Comment (ihor-romaniuk) [reply src/library-authoring/list-libraries/LibraryListPage.jsx]: added

File: src/vendor/studio.scss (hunk 2 above)
Comment (ihor-romaniuk) [reply src/vendor/studio.scss]: replaced

File: src/vendor/studio.scss (hunk 4 above)
Comment (ihor-romaniuk) [reply src/vendor/studio.scss]: replaced

File: src/vendor/studio.scss (hunk 3 above)
Comment (ihor-romaniuk) [reply src/vendor/studio.scss]: replaced

**Review by dyudyunov — APPROVED** [review]

**Review by UvgenGen — APPROVED** [review]

File: src/library-authoring/create-library/LibraryCreatePage.jsx [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+            activeLabel={intl.formatMessage(messages['library.form.breadcrumbs.current'])}
+            clickHandler={this.handleClickBreadcrumbs}
+            links={[
+              { label: intl.formatMessage(messages['library.form.breadcrumbs.home']), url: config.STUDIO_BASE_URL },
+              { label: intl.formatMessage(messages['library.form.breadcrumbs.list']), url: ROUTES.List.HOME }
+            ]}
+          />
+          <div className="wrapper-mast wrapper">
+            <header className="has-actions">
+              <h1 className="page-header h2">{intl.formatMessage(messages['library.form.create.library'])}</h1>
```
Comment (ruzniaievdm) [inline src/library-authoring/create-library/LibraryCreatePage.jsx]: Why do you use h1 tag with .h2?

Comment (ihor-romaniuk) [reply src/library-authoring/create-library/LibraryCreatePage.jsx]: fixed, thanks

File: src/library-authoring/common/specs/OrganizationDropdown.spec.jsx [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+  });
+
+  return map;
+};
+
+describe('common/OrganizationDropdown.jsx', () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
```
Comment (monteri) [inline src/library-authoring/common/specs/OrganizationDropdown.spec.jsx]: It is better to put `const props = ...` in the start of the file and remove `beforeEach` in this case

File: src/library-authoring/create-library/specs/LibraryCreatePage.spec.jsx [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+
+describe('create-library/LibraryCreatePage.jsx', () => {
+  const config = { STUDIO_BASE_URL: 'STUDIO_BASE_URL' };
+  let props;
+  let mockResetForm;
+  let mockCreateLibrary;
+  let mockFetchOrganizations;
+
+  beforeEach(() => {
+    mockResetForm = jest.fn();
```
Comment (monteri) [inline src/library-authoring/create-library/specs/LibraryCreatePage.spec.jsx]: The same here, it is possible to define all these variables in the start of the file

**Pull request merged**